docs/courses/csintro3/about/start.md

```diff
@@ -21,10 +21,13 @@ and blocks that are unique to game development.
 The following tutorials may also be useful in learning about concepts that
 you might not have seen elsewhere:
 
-* [Eat the Doughnut](/#tutorial:/tutorials/eat-the-doughnut)
+* [Chase the Pizza](/#tutorial:/tutorials/chase-the-pizza)
 * [Simple Maze](/#tutorial:/tutorials/simple-maze)
-* [Star Field](/#tutorial:/tutorials/star-field)
 * [Happy Flower](/#tutorial:/tutorials/happy-flower)
+* [Lemon Leak](/#tutorial:/tutorials/lemon-leak)
```

Comment:
Maybe a few of the game design concepts here, in case theyre really starting from scratch?